app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  after_create :update_likes_count
+
+  def update_likes_count
+    post = Post.find(post_id)
+    post.increment(:likes_counter)
+  end
```

Comment:
- [ ] Same issue as discussed in the Comment model up there. Please, let's make this method private and refactor as necessary.

  ```suggestion
  private
  
  def update_likes_count
    post.increment!(:likes_counter)
  end
  ```

Reply:
This issue is well resolved! ✔️